README.md

```diff
@@ -7,14 +7,30 @@ A Jetpack-powered companion app for WooCommerce.
 
 ## Build Instructions
 
-- Download Xcode
-  - At the moment *WooCommerce for iOS* uses Swift 5 and requires Xcode 10.2 or newer. Previous versions of Xcode can be [downloaded from Apple](https://developer.apple.com/downloads/index.action)
-- Clone project by `git clone https://github.com/woocommerce/woocommerce-ios.git` in the folder of your preference
-- Enter the project directory by `cd woocommerce-ios`
-- Install the third party dependencies and tools required to run the project
-  - We use a few tools to help with development. To install or update the required dependencies, run the follow command on the command line: `bundle exec pod install`
-  - In some cases, you may also have to: `bundle install`
-- Open the project by double clicking on `WooCommerce.xcworkspace` file, or launching Xcode and choose File > Open and browse to `WooCommerce.xcworkspace`
+1. Download Xcode
+
+    At the moment *WooCommerce for iOS* uses Swift 5 and requires Xcode 10.2 or newer. Previous versions of Xcode can be [downloaded from Apple](https://developer.apple.com/downloads/index.action)
```

Comment:
nit: I don't think our project can be built with Xcode 10.2 anymore (just tried building with Xcode 10.2.1 and it had build errors), maybe we could update the Xcode version to Xcode 11+ or 11.2+? The CI uses Xcode 11.2.1 now:

https://github.com/woocommerce/woocommerce-ios/blob/a578b8808833a45222edbf762ccab8fe3e60b8da/.circleci/config.yml#L21